app/controllers/api/events_controller.rb

```diff
@@ -28,6 +33,13 @@ def sequence_id_for_comment(collection)
   # we always want to serialize out events in the events controller
   alias :events_to_serialize :resources_to_serialize
 
+  def respond_with_reader(scope: default_scope, serializer: DiscussionReaderSerializer, root: :discussions)
+    render json: Array(DiscussionReader.for_model(resource.discussion, current_user)),
```

Comment:
Here we serialize out JSON like this:
```
{ discussions: [
  id: #
  last_read_at: (time)
  last_read_sequence_id: # 
]}
```
which will update the discussion on the client with the new reader values. It's one of the only times we respond with a resource that _isn't_ the controller resource, but I think the tradeoff of having an endpoint that we just send an id to (rather than a discussion id and sequence id) is worth it.